cmd/podman/main.go

```diff
@@ -36,6 +38,20 @@ func main() {
 		return
 	}
 
+	if filepath.Base(os.Args[0]) == registry.PodmanSh ||
+		(len(os.Args[0]) > 0 && filepath.Base(os.Args[0][1:]) == registry.PodmanSh) {
+		shell := strings.TrimPrefix(os.Args[0], "-")
+		args := []string{shell, "exec", "-i", "--wait", "10"}
```

Comment:
@rhatdan wdyt?

Reply:
@bachradsusi guess we could leave this for a followup

Reply:
SGTM, but let's do it in a secondary PR, and also do it in containers.conf.  It would be nice to some how put out a warning, but I am not sure how we could get this to a user.  Perhaps at X amount of time say 5 seconds, we inform the user that maybe a container image is being pulled.